src/server/game/Handlers/SpellHandler.cpp

```diff
@@ -529,22 +516,36 @@ void WorldSession::HandlePetCancelAuraOpcode(WorldPacket& recvPacket)
         return;
     }
 
-    pet->RemoveOwnedAura(spellId, ObjectGuid::Empty, 0, AURA_REMOVE_BY_CANCEL);
+    pet->RemoveOwnedAura(packet.SpellID, ObjectGuid::Empty, 0, AURA_REMOVE_BY_CANCEL);
 }
 
-void WorldSession::HandleCancelGrowthAuraOpcode(WorldPacket& /*recvPacket*/) { }
+void WorldSession::HandleCancelGrowthAuraOpcode(WorldPackets::Spells::CancelGrowthAura& /*cancelGrowthAura*/)
+{
+    _player->RemoveAurasByType(SPELL_AURA_MOD_SCALE, [](AuraApplication const* aurApp)
```

Comment:
is this supposed to remove all auras or only last or another condition ?

Reply:
It will remove all auras with type SPELL_AURA_MOD_SCALE that don't have SPELL_ATTR0_CANT_CANCEL and not passive or negative

Reply:
@jackpoz if you have any issues with the current code fix them in master branch (because thats where the implementation comes from) but absolutely do not make any stealth fixes in port commits (because that in turn makes porting in the other direction impossible)

Reply:
@Shauren this PR doesn't mention anywhere that any code is ported from master branch. It would be nice to have it in the PR description and maybe have the -x parameter when cherry-picking the commits.